src/_components/skip.css

```diff
@@ -1,12 +1,11 @@
 .skip {
-	position: absolute;
-	right: 100%;
-	color: var(--main-color);
+		position: absolute;
+		right: 100%;
+		color: var(--main-color);
+}
 
-	&:focus {
+.skip:focus {
```

Comment:
If you're using PostCSS and esbuild anyway, why not have either [convert nested CSS](https://prepitaph.org/snippets/css-nesting-esbuild/) during deployment?

Reply:
Good point, I will take a look

Reply:
Noted as https://github.com/faucet-pipeline/docs/issues/94